.azure-pipelines/MacOS-CI.yml

```diff
@@ -119,4 +119,9 @@ jobs:
         exit 1
       fi
 
+      git ls-files --others --error-unmatch --exclude-standard
+      if [ $? -ne 0 ]; then
+        echo "git diff for test generation, because there might be some untrack files. Please upload them as well"
```

Comment:
nit: may be improve the message a bit: "Some test-generated files not included in PR. Did you forget to add any test-generated files?"

Reply:
Looks better. I have updated it. PTAL. Thanks!